osbs/utils.py

```diff
@@ -52,6 +52,10 @@
 ClonedRepoData = namedtuple('ClonedRepoData', ['repo_path', 'commit_id', 'commit_depth'])
 
 
+class OsbsCommitNotFound(OsbsException):
```

Comment:
Please add docstring

Reply:
Also it should't inherit from `OsbsException` as  this exception should never reach user. Use just `Exception`